js/src/dapps/registry/Accounts/accounts.js

```diff
@@ -61,20 +65,27 @@ export default class Accounts extends Component {
   }
 
   renderAccount = (account) => {
-    const { all, selected } = this.props;
+    const { selected } = this.props;
     const isSelected = selected && selected.address === account.address;
 
     return (
       <MenuItem
         key={ account.address } value={ account.address }
         checked={ isSelected } insetChildren={ !isSelected }
       >
-        { renderAddress(account.address, all, {}) }
+        <Address address={ account.address } />
       </MenuItem>
     );
   };
 
   onAccountSelect = (e, address) => {
-    this.props.actions.select(address);
+    this.props.select(address);
   };
 }
+
+export default connect(
+  // mapStateToProps
+  (state) => state.accounts,
```

Comment:
If we have to comment this to state intent, something is wrong, the code should just be self-explanatory (if the reader gets confused, it means we are not making the writing clear enough - a reason why in the rest of the app the functions are properly named)